src/quo2/components/drawers/bottom_actions/view.cljs

```diff
@@ -0,0 +1,39 @@
+(ns quo2.components.drawers.bottom-actions.view
+  (:require [quo2.components.buttons.button.view :as button]
+            [quo2.components.drawers.bottom-actions.style :as style]
+            [quo2.theme :as theme]
```

Comment:
we import this as `[quo2.theme :as quo.theme]` to avoid clashes with prop `theme`